sdk/monitor/azure-monitor-opentelemetry-autoconfigure/README.md

```diff
@@ -0,0 +1,199 @@
+    applicationLogic();
+} catch (Throwable t) {
+    span.recordException(t);
+    throw t;
+} finally {
+    span.end();
+}
+```
+The following example demonstrates how to add a span processor to the OpenTelemetry SDK autoconfiguration.
+
+```java readme-sample-span-processor
+private static final AttributeKey<String> ATTRIBUTE_KEY = AttributeKey.stringKey("attributeKey");
+
+public void spanProcessor() {
+    AutoConfiguredOpenTelemetrySdkBuilder sdkBuilder = AutoConfiguredOpenTelemetrySdk.builder();
+
+    AzureMonitorAutoConfigure.customize(sdkBuilder);
+
+    SpanProcessor spanProcessor = new SpanProcessor() {
+
+        @Override
+        public void onStart(Context context, ReadWriteSpan span) {
+            span.setAttribute(ATTRIBUTE_KEY, "attributeValue");
+        }
+
+        @Override
+        public boolean isStartRequired() {
+            return true;
+        }
+
+        @Override
+        public void onEnd(ReadableSpan readableSpan) {
+        }
+
+        @Override
+        public boolean isEndRequired() {
+            return false;
+        }
+    };
+
+    sdkBuilder.addTracerProviderCustomizer(
+        (sdkTracerProviderBuilder, configProperties) -> sdkTracerProviderBuilder
+            .addSpanProcessor(spanProcessor));
+}
+```
+More advanced examples with OpenTelemetry APIs:
+* [Advanced examples - 1][advanced_examples_1]
+* [Advanced examples - 2][advanced_examples_2]
+* [Event Hubs example][event_hubs_example]
+
+## Key concepts
+
+Some key concepts for the Azure Monitor exporter include:
+
+* [OpenTelemetry][opentelemetry_spec]: OpenTelemetry is a set of libraries used to collect and export telemetry data
+ (metrics, logs, and traces) for analysis in order to understand your software's performance and behavior.
+
+* [Instrumentation][instrumentation_library]: The ability to call the OpenTelemetry API directly by any application is
+ facilitated by instrumentation. A library that enables OpenTelemetry observability for another library is called an Instrumentation Library.
+
+* [Trace][trace_concept]: Trace refers to distributed tracing. It can be thought of as a directed acyclic graph (DAG) of Spans, where the edges between Spans are defined as parent/child relationship.
+
+* [Tracer Provider][tracer_provider]: Provides a `Tracer` for use by the given instrumentation library.
+
+* [Span Processor][span_processor]: A span processor allows hooks for SDK's `Span` start and end method invocations. Follow the link for more information.
+
+* [Sampling][sampler_ref]: Sampling is a mechanism to control the noise and overhead introduced by OpenTelemetry by reducing the number of samples of traces collected and sent to the backend.
+
+For more information on the OpenTelemetry project, please review the [OpenTelemetry Specifications][opentelemetry_specification].
+
+
+## Troubleshooting
+
+### Enabling Logging
+
+Azure SDKs for Java offer a consistent logging story to help aid in troubleshooting application errors and expedite
```

Comment:
its probably been there for exporter, but I'm pretty sure this section is not accurate for exporter/distro and otel, it'd be nice to add a link to java otel troubleshooting.